src/Chapter09.Tests/Listing09.02.Tests.cs

```diff
@@ -1,15 +1,29 @@
 ﻿namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter09.Listing09_02.Tests;
 
 [TestClass]
-public class AngleProgramTests
+public class AngleTests
 {
     [TestMethod]
-    public void Create_Angle_Success()
+    public void Create_AngleArray_ElementsAreUninitialized()
     {
-        string expected =
-            "Angle { Degrees = 90, Minutes = 0, Seconds = 0, Name =  }";
-        IntelliTect.TestTools.Console.ConsoleAssert.Expect(
-            expected, Program.Main);
+        Angle angle = default;
+        Angle[] angles = new Angle[1];
+        Assert.AreEqual(angle, angles[0]);
     }
 
+    [TestMethod]
+    public void Equals_CopyWithSameDegreesMinutesSeconds_AreEqual()
+    {
+        Angle angle1 = new(0, 0, 0);
+        Angle angle2 = new(0, 0, 0);
```

Comment:
This works, but a slightly better option is to avoid using default values (such as 0 for numeric types) when you want to test their values since it is impossible to tell if the constructor is using the value you passed in or if the values are just defaulting to 0.

```suggestion
        Angle angle1 = new(1, 2, 3);
        Angle angle2 = new(1, 2, 3);
```